pkg/config/agent/config.go

```diff
@@ -260,6 +237,46 @@ type NodePortLocalConfig struct {
 	PortRange string `yaml:"portRange,omitempty"`
 }
 
+type FlowExporterConfig struct {
+	// Enable FlowExporter, a feature used to export polled conntrack connections as IPFIX flow records from each agent
```

Comment:
same comment as above. Please use fewer characters per line to match other field comments.